dashboard/serializers.py

```diff
@@ -49,14 +49,27 @@ def serialize_enrollment(cls, mmtrack, course_run):
         payment_status = cls.PAID_STATUS if has_paid else cls.UNPAID_STATUS
 
         final_grade = mmtrack.get_final_grades_for_course(course_run.course).first()
-        semester = cls.serialize_semester(course_run)
         return {
             'final_grade': final_grade.grade_percent if final_grade else None,
-            'semester': semester,
             'course_title': course_title,
             'payment_status': payment_status,
         }
 
+    @classmethod
+    def semesters_enrolled(cls, mmtrack):
```

Comment:
Should be called `serialize_semesters_enrolled` or `serialize_enrolled_semesters`